synapse/util/ldap_auth_provider.py

```diff
@@ -237,13 +237,14 @@ def _ldap_simple_bind(self, server, localpart, password):
                 base=self.ldap_base
             )
             conn = ldap3.Connection(server, bind_dn, password,
-                                    authentication=ldap3.AUTH_SIMPLE)
+                                    authentication=ldap3.SIMPLE)
```

Comment:
This was addressed in the external module. Whether `SIMPLE` or `AUTH_SIMPLE` works for you is dependant on your ldap3 version.